website/docs/docs/deploy/job-notifications.md

```diff
@@ -1,33 +1,83 @@
 
-2. Select **Edit** to begin editing the **Email Notifications** settings.
-    - **As a user:** Choose the Notification type (Succeeds, Fails, or Is Cancelled) for each Job you want to receive notifications for.
+    <Lightbox src="/img/docs/deploy/example-notification-external-email.png" width="50%" title="Example of the Notification email dropdown"/>
 
-    - **As an admin:**  Under **Configure notifications for**, use the dropdown to select one or more users you'd like to set notifications for. If you only see your own name, then you might not have admin privileges. <br /><br />
-    Choose the Notification type (Succeeds, Fails, or Is Cancelled) for each Job you want them to receive notifications for.
+1. Select the **Environment** (in the dropdown) for the jobs you want to receive notifications about. 
 
-3. Click **Save**.
+1. Click **Edit** to configure the email notification settings. Choose one or more of the run statuses (**Succeeds**, **Fails**, **Is Canceled**) for each job you want to receive notifications about.
 
-    <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/email-notifications.png" width="75%" title="Configuring Email Notifications"/>
+1. When you're done with the settings, click **Save**.
 
-### Slack
+    If there is another person you also want to send the notifications to, as an account admin, you can select another **Notification email** from the dropdown, **Edit** the job notification settings, and **Save** the changes.
+    
+    To set up alerts on jobs from a different environment, select another **Environment** from the dropdown, **Edit** those job notification settings, and **Save** the changes. 
 
-<Snippet path="slack-notifications-config-steps" />
+    <Lightbox src="/img/docs/deploy/example-email-notification-settings-page.png" width="100%" title="Example of the Email notifications page"/>
+
+### Unsubscribe from email notifications
+1. From the gear menu in the upper right corner, choose **Notification settings**.
+1. On the **Email notifications** page, click **Unsubscribe from all email notifications**. 
+
+## Slack notifications
+
+You can receive Slack alerts about jobs by setting up the Slack integration, then configuring the dbt Cloud Slack notification settings.
+
+:::note 
+Any account admin can edit the Slack notifications but they'll be limited to configuring the channels that their Slack user has access to. If there has been a change in user roles or Slack permissions where you no longer have access to edit a configured Slack channel, please [contact support](mailto:support@getdbt.com) for assistance. 
+:::
+
+### Prerequisites 
+- You must be an administrator of the Slack workspace. 
+- You must be an account admin to configure Slack notifications in dbt Cloud. For more details, refer to [Users and licenses](/docs/cloud/manage-access/seats-and-users).
+
+### Set up the Slack integration
```

Comment:
Could potentially save space here and better serve customers by linking to the existing FAQ. Maybe even a standalone page in the setup. Linking in the pre-reqs could save some space as well.

Reply:
for a better reading experience, i actually consolidated this information so it's in one place instead of various places. in this pr, i removed the FAQ and added a redirect to this page.